pallets/staking/src/lib.rs

```diff
@@ -2644,6 +2646,19 @@ decl_module! {
                     .map(|p| p.intended_count = new_intended_count)
             })
         }
+
+
+        /// Declare no desire to either validate or nominate.
+        ///
+        /// Effects will be felt at the beginning of the next era.
+        ///
+        /// The dispatch origin for this call must be a GC.
+        /// And, it can be only called when [`EraElectionStatus`] is `Closed`.
```

Comment:
Please update the description to match this new extrinsic.  Also make sure the docs match the standard style that we use listing the parameters and errors.